### DISCUSSION TITLE
google chrome

### DISCUSSION THREAD

**Discussion by larrygtr** [community/community#60401] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Google chrome only works with  blank anchor links when you target <a href="#">Blog</a> with a pseudo link in css a: link. It does notwork for actual anchor links with real links where # is an actual href, like "https://github.com/orgs/community/discussions/new?category=feed" . I had to change my default Web Browser to Brave. Does anyone know why?

**Comment by hamidrezamd1380** [comment]
https://realit.ir/

**Comment by jgbenine** [comment]
Whenever we use the <a> tag, we need to link a text or some type of content in that link it supports external and internal directing links, it makes no sense to use blank links or without content to direct the user to some place.

If you are using a # in no href it will just take you to the top of the page and if you are leaving this property empty it will just refresh your page at the same location.
